skelebot/systems/execution/commandBuilder.py

```diff
@@ -25,13 +25,23 @@ def build(config, job, args, native=False):
 
     return COMMAND.format(extCommand=EXT_COMMAND[ext], file=job.source, args=arguments, params=params, pre=pre, post=post)
 
-def buildArgs(jobArgs, args, template):
+def buildArgs(argParams, args):
     argString = ""
-    if (jobArgs != None):
-        for arg in jobArgs:
-            value = args[arg.name] if (arg.name in args) else arg.default
-            value = "" if (arg.isBoolean) else value
+    if (argParams != None):
+        for arg in argParams:
+            value = args[arg.name] if (arg.name in args) else None
             if (value != None):
-                argString += template.format(name=arg.name, value=value)
+                argString += " {}".format(value)
 
     return argString
+
+def buildParams(jobParams, args):
+    paramString = ""
+    if (jobParams != None):
+        for param in jobParams:
+            value = args[param.name] if (param.name in args) else param.default
```

Comment:
Similar to above, this line can be rewritten to: `value = args.get(param.name, param.default)`